examples/platforms/efr32mg21/radio.c

```diff
@@ -45,7 +45,6 @@
 #include "utils/soft_source_match_table.h"
```

Comment:
There seems to be a number of changes from the existing `efr32/radio.c` applied to `efr32mg12/radio.c` that are not reflected here in `efr32mg21/radio.c`. Is that intentional?

Reply:
Yes it is. There is another PR in the works that adds the sleepy demo to MG21 and that PR will have an updated radio driver.